apps/manifest.go

```diff
@@ -98,7 +98,44 @@ type Manifest struct {
 	// Cloud;
 	// - define path->function mappings, aka "routes". The function with the
 	// path matching as the longest prefix is used to handle a Call request.
-	AWSLambda []AWSLambdaFunction `json:"aws_lambda,omitempty"`
+	AWSLambda []AWSLambda `json:"aws_lambda,omitempty"`
+}
+
+// AWSLambda describes a distinct AWS Lambda function defined by the app, and
```

Comment:
0/5 Makes it easier to have it here rather than try to abstract type-specific structs; may still come as a separate PR. Other AWS-related types etc are now in `upstrem/upaws`